middleware/hcaptcha/config.go

```diff
@@ -0,0 +1,33 @@
+package hcaptcha
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/gofiber/fiber/v3"
+)
+
+const DefaultSiteVerifyURL = "https://api.hcaptcha.com/siteverify"
```

Comment:
This requires a subscription, it shouldnt be in `gofiber/fiber`. Maybe in `gofiber/contrib`